GGSmtp/GGSmtp.Tests/DataResponderTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using NUnit.Framework;
+using Simple.MailServer.Smtp.Config;
+using Moq;
+using Simple.MailServer.Smtp;
+using System.IO;
+using Simple.MailServer;
+using System.Text;
+
+namespace GGSmtp.Tests
+{
+    [TestFixture]
+    public class DataResponderTests
+    {
+        [Test(Author = "Dor", Description = "Checks the DefaultDataResponder", TestOf = typeof(DefaultDataResponder)]
+        public void TestAttachmentSend()
+        {
+            string[] lines = File.ReadAllLines(@"C:\Temp\Mail\2016-05-15_113132_746.eml");
```

Comment:
Accessing local files is problematic.
This test will fail on any new PC.
I'd hard-code the lines. !fix